src/cli/index.js

```diff
@@ -62,6 +62,10 @@ commander.option(
   '--no-emoji',
   'disable emoji in output',
 );
+commander.option(
+  '--disable-progress',
```

Comment:
The CLI convention is to use `no-` for negated flags, so this should be `--no-progress`.